packages/validation/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env sh
+
+rm -rf dist/ lionweb-core-*.tgz
```

Comment:
Last term should be "`lionweb-validation-*.tgz`".

Reply:
Is this txz files created somewhere? I don't see it appear when I build the project.

Reply:
They're created by `npm pack`, so it's not relevant for the normal flow, but I like to keep cleaning the repo and the contents of `.npmignore` somewhat in sync.